libraries/ms-common/inc/x86/uart_mcu.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
 typedef enum {
-  NUM_UART_PORTS = 0,
+  UART_PORT_1 = 0,
```

Comment:
We should maybe look into an alternate method of doing these in future, having to make the ports the same across x86 and stm32f0xx even if they aren't the same seems kind of useless. Not a blocker just something to keep in mind.

Reply:
My original intention was for projects to have config files per platform that could have defines. So instead of a project directly using `UART_PORT_1`, it would use `PROJECT_FOO_UART_PORT` or something similar.